src/app/components/admin/audio-recordings/audio-recording.schema.json

```diff
@@ -2,170 +2,149 @@
   "fields": [
     {
       "key": "id",
-      "type": "input",
+      "hide": true,
```

Comment:
maybe what you mean to express here is `readOnly`?

Reply:
I rarely want to use readOnly, if users can't edit a field it shouldn't be shown in the form (unless you can think of an example where the additional information would help users?). The hidden fields are just for the admin detail pages where i want to show all of the model parameters

Reply:
well you probably still want readOnly.

as for admin pages, why would I not want to see all information?

Reply:
These are not hidden from the admin view, only the form. Most of this schema is defining how the form will be displayed and has no affect on the DetailsViewComponent